app/models/concerns/exhibit_extension.rb

```diff
@@ -17,4 +19,11 @@ def required_viewer
     return viewer if viewer.present?
     Viewer.create(exhibit: self)
   end
+
+  private
+
+  def send_publish_state_change_notification
+    return unless saved_changes.keys.include?('published')
+    SendPublishStateChangeNotificationJob.perform_later(exhibit: self, published: published)
```

Comment:
This `published` variable is a little confusing to me. `Spotlight::Exhibit` has it as an ActiveRecord::Relation (i.e., `scope :published, -> { where(published: true) }`). In the Job code, you're just using it as a boolean flag. Maybe use `published.present?` here to indicate that it's just a flag? Or am I misunderstanding this code?

Reply:
`#published` on an instance of an Exhibit is a boolean while `#published` on the Exhibit class is a scope (I do understand that the scope and instance method being named the same thing is a bit confusing).

```ruby
Spotlight::Exhibit.published => ActiveRecord::Relation
Spotlight::Exhibit.new.published => Boolean
```

I'm not 100% sure saying effectively saying `true.present?` vs `false.present?` makes things any clearer here?

Reply:
ok that makes sense now. thanks.